lib/internal/util.js

```diff
@@ -487,14 +488,18 @@ function isInsideNodeModules() {
     // Use `runInNewContext()` to get something tamper-proof and
     // side-effect-free. Since this is currently only used for a deprecated API,
     // the perf implications should be okay.
-    getStructuredStack = runInNewContext(`(function() {
-      try { Error.stackTraceLimit = Infinity; } catch {}
-      return function structuredStack() {
-        const e = new Error();
-        overrideStackTrace.set(e, (err, trace) => trace);
-        return e.stack;
-      };
-    })()`, { overrideStackTrace }, { filename: 'structured-stack' });
+    getStructuredStack = runInNewContext(
+      `(function() {
+        try { Error.stackTraceLimit = Infinity; } catch {}
+        return function structuredStack() {
+          const e = new Error();
+          overrideStackTrace.set(e, (err, trace) => trace);
+          return e.stack;
+        };
+      })()`,
```

Comment:
What's the change in here? Is it only indentation? I couldn't figure out the difference, sorry.

Reply:
yeah, only intendation. I can revert if you like.

Reply:
appreciate it, if you could. i prefer the new change but we don't have any mechanism to persist this change (yet)